### DISCUSSION TITLE
"Projects cannot have more than 1200 items"

### DISCUSSION THREAD

**Discussion by simonw** [community/community#20550] [Projects and Issues]
Just got this error message trying to add a new item to a project:

![image](https://user-images.githubusercontent.com/9599/179065360-55ccf8a8-0773-4ba9-9bdd-46106d0a3cf3.png)

I've been using Projects to combine TODO items from all of my 100+ active GitHub repositories... but this limit means it's not going to work for that, at all.

I'd really appreciate it if this limit could be lifted, and ideally could eventually be removed entirely (or bumped up to ~10,000 where it's much less likely to affect how I'm using Projects).

**Comment by simonw** [comment]
I'm trying to archive items now - I have about 900 that are in my "Done" status where archiving them would make sense.

Problem: although it says that the "e" key should work as a keyboard shortcut for archiving, nothing happens when I hit that key. I have to click the arrow menu, then click archive, then click "yes" on the "Are you sure" screen... for every one of 900 items!

**Reply by ekroon** [reply]
In table overview it is possible to click a row and then shift click to select everything in between. It is quite finicky and it will need a couple of tries, but after that it is possible to archive everything in one step. 

Alternatively: In a board overview it is possible to archive a full column (except the "No Status" column), by using the `...` in the top right corner of the column.

**Reply by simonw** [reply]
I could not figure out how to do the shift-click select in table view - but your board tip has solved my problem for me, thanks!

**Comment by ianwilliams1** [comment]
Related discussion: [Unable to add more than 1200 items to the project #9678](https://github.com/community/community/discussions/9678)
